R/backport_linter.R

```diff
@@ -42,6 +42,9 @@ backport_linter <- function(r_version = getRversion(), except = character()) {
 
   backport_blacklist <- backports[r_version < R_system_version(names(backports))]
   backport_blacklist <- lapply(backport_blacklist, setdiff, except)
+  backport_funs <- unlist(backport_blacklist)
+  backport_versions <- rep(names(backport_blacklist), times = lengths(backport_blacklist))
+  backport_index <- stats::setNames(backport_versions, backport_funs)
```

Comment:
why keep `backport_versions` and `backport_index` separately? Why not just `names(backport_versions) <- backport_funs`?

Reply:
We are using `backport_index` in the code.
I can shorten the code that produces it if you like?